src/components/FloatingHelper/FloatingHelper.st.css

```diff
@@ -0,0 +1,54 @@
+:import {
+  -st-from: "wix-ui-core/dist/src/baseComponents/Popover/Popover.st.css";
+  -st-default: Popover;
+}
+
+:import {
+  -st-from: "../../palette.st.css";
+  -st-named: brandSecondary, backgroundSecondary
+}
+
+:import {
+  -st-from: "../../typography.st.css";
+  -st-named: fontRoman;
+}
+
+:vars {
+  /* TODO: create and use a fade function */
```

Comment:
You have a lot of todos. Maybe create an issue which accumulates them all?

Reply:
Its here:
https://core3.leankit.com/card/656415410